modules/operator/scripts/install_helm.template.sh

```diff
@@ -2,16 +2,16 @@
 # Copyright 2020, Oracle Corporation and/or affiliates.  
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
 
-echo 'installing helm'
+echo 'Installing helm'
 
-wget https://get.helm.sh/helm-v${helm_version}-linux-amd64.tar.gz 2> /dev/null
+wget https://get.helm.sh/helm-v"${helm_version}"-linux-amd64.tar.gz 2> /dev/null
```

Comment:
Why are we downloading Helm instead of installing the Helm module?

Reply:
I broke that change out to a different PR #3 - I wanted this PR to tidy up code but make no changes to logic.

Reply:
Yep, sorry. I only saw that PR after reviewing this one. :)